### ISSUE TITLE
[bazel.build] Problem with /install

### ISSUE THREAD

**Issue by Kve-sudo** [bazelbuild/bazel#20892]
### Page link:

https://bazel.build/install

### Problem description (include actual vs expected text, if applicable):

tengo problemas con crosh ingreseo el comando shell y me dice que hay un error

### Where do you see this issue? (include link to specific section of the page, if applicable)

crosh

### Any other information you'd like to share?

ya he reiniciado mi chromeboook varias vecse he seguido al pie de la letra donde he leido y visto varios videos de desde aqui de github he copiado toda la informacion y me dice lo mismo

**Comment by meisterT** [comment]
How is your problem related to Bazel? Also please edit the first comment and describe in English what the problem is.

**Comment by Kve-sudo** [comment]
The problem is that I want to install Ubuntu, I open crosh and enter the shell command. After this I have entered several commands and nothing tells me to adjust the configuration, I also did it and nothing tells me anything about privileges.  I enter help to give me the commands I need and it responds that help or advance help is not a command.  I have entered sudo sh -e /Downloads/crouton/ -y unity and several more similar to the latter and nothing does not let me is all.

**Comment by Kve-sudo** [comment]
The problem is that I want to install Ubuntu, I open crosh and enter the shell command. After this I have entered several commands and nothing tells me to adjust the configuration, I also did it and nothing tells me anything about privileges.  I enter help to give me the commands I need and it responds that help or advance help is not a command.  I have entered sudo sh -e /Downloads/crouton/ -y unity and several more similar to the latter and nothing does not let me is all.

**Comment by Kve-sudo** [comment]
The problem is that I want to install Ubuntu, I open crosh and enter the shell command. After this I have entered several commands and nothing tells me to adjust the configuration, I also did it and nothing tells me anything about privileges.  I enter help to give me the commands I need and it responds that help or advance help is not a command.  I have entered sudo sh -e /Downloads/crouton/ -y unity and several more similar to the latter and nothing does not let me is all.

**Comment by Kve-sudo** [comment]
The problem is that I want to install Ubuntu, I open crosh and enter the shell command. After this I have entered several commands and nothing tells me to adjust the configuration, I also did it and nothing tells me anything about privileges.  I enter help to give me the commands I need and it responds that help or advance help is not a command.  I have entered sudo sh -e /Downloads/crouton/ -y unity and several more similar to the latter and nothing does not let me is all.
